test/test_examples_2d.jl

```diff
@@ -66,12 +66,45 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
+    l2   = [3.6343269600298813e-6, 0.003211138535470753, 0.0032111483996635176, 0.004545716490663252],
+    linf = [7.900701569263369e-5, 0.03056161620998943, 0.030502702817406968, 0.04287693324734221])
+  end
+
+  @testset "elixir_mhd_alfven_wave.jl" begin
+  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_alfven_wave.jl"),
+    l2   = [0.0001113433445427984, 5.8797911730861035e-6, 5.879791173070769e-6, 8.432663153049379e-6, 1.2944133252643067e-6, 1.2228196429171974e-6, 1.2228196429188102e-6, 1.8293483299776387e-6, 8.098813436751192e-7],
+    linf = [0.00026789134725313524, 1.623828993739529e-5, 1.6238289936978956e-5, 2.736135667090467e-5, 5.328020565742442e-6, 8.130112221338415e-6, 8.13011222156046e-6, 1.2125090631517899e-5, 4.179925589405737e-6])
+  end
+
+  @testset "elixir_mhd_alfven_wave_mortar.jl" begin
+  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_alfven_wave_mortar.jl"),
+    l2   = [4.610813220207719e-6, 1.6899574535067422e-6, 1.6210569114291013e-6, 1.6996243959894436e-6, 1.4864362277675424e-6, 1.387782728212359e-6, 1.3413790178828178e-6, 1.7156734550669053e-6, 9.813890556562632e-7],
+    linf = [3.522563561497716e-5, 1.53507840691125e-5, 1.4265094405635503e-5, 1.4421769785291638e-5, 7.74418952287359e-6, 1.0188712513015119e-5, 9.862796218396142e-6, 1.601803921538747e-5, 5.563694484229304e-6],
+    tspan = (0.0, 1.0))
+  end
+
   @testset "elixir_mhd_ec.jl" begin
   test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_ec.jl"),
     l2   = [0.03628315925311581, 0.04301306535453907, 0.042998910996002976, 0.025746791646914315, 0.1620587870592711, 0.01745580631201365, 0.01745656644392971, 0.02688212902288343, 0.00014263322984147517],
     linf = [0.23504901239438747, 0.31563591777956146, 0.3094412744514615, 0.21177505529310434, 0.9738775041875032, 0.09120517132559702, 0.0919645047337756, 0.15691668358334432, 0.0035581030835232378])
   end
 
+  @testset "elixir_mhd_orszag_tang_amr.jl" begin
+  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_orszag_tang_amr.jl"),
+    l2   = [0.10792892784381206, 0.20179133761722504, 0.22962184589189918, 0.0, 0.2993794211703735, 0.1570642713959375, 0.24295137209226844, 0.0, 0.012333314953338195],
+    linf = [0.5613440997713406, 0.5101525577854369, 0.6592905328573747, 0.0, 0.9883261765401286, 0.401691133525005, 0.6750421877831418, 0.0, 0.20250593792711205],
+    tspan = (0.0, 0.06))
+  end
+
+  @testset "elixir_mhd_rotor_amr.jl" begin
+  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_rotor_amr.jl"),
+    l2   = [0.3706256586052198, 0.8397913575257775, 0.8061150883165829, 0.0, 0.9158524987760742, 0.11469774220979392, 0.14167686867748344, 0.0, 0.01843234514008023],
+    linf = [4.909133664840393, 9.429512581671567, 7.668678084481505, 0.0, 10.777238495921754, 1.294018631465027, 1.4569385394302714, 0.0, 0.2734224559555755],
+    tspan = (0.0, 0.02))
+  end
+
   @testset "elixir_euler_gravity_jeans_instability.jl" begin
```

Comment:
```suggestion
  end
  
  @test_skip test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_blast_wave.jl"), tspan=(0.0, 1.0e-4))

  @testset "elixir_euler_gravity_jeans_instability.jl" begin
```

Reply:
okay, I thought I could set a small enough time span such that only one step is taken because I saw in source that the parameter `n_steps_max` is only valid in run and will be removed in Taal.

Reply:
`n_steps_max` will not be included directly in our part of Taal, since the ODE solvers are now contained in OrdinaryDiffEq.jl. We can achieve the same result by passing the additional keyword argument `maxiters` to `solve(ode, ...)`, e.g.
```julia
sol = solve(ode, CarpenterKennedy2N54(williamson_condition=false), dt=stepsize_callback(ode),
            save_everystep=false, callback=callbacks, maxiters=1);
```
see https://diffeq.sciml.ai/v6.8/basics/common_solver_opts/#Miscellaneous-1. I've opened an issue to make this easily available via `trixi_include` (https://github.com/trixi-framework/Trixi.jl/issues/224).